doc/Guide to Contributing to Sia.md

```diff
@@ -0,0 +1,253 @@
+# Contributing to Sia
+
+#### Table of Contents
+* [Get started with Go](#go)
+  * [Install Go](#install-go)
+  * [Learn Go]("learn-go")
+* [Build Sia](#build-sia)
+* [Contribute to the codebase](#contribute)
+  * [Set up git](#setup-git)
+  * [Fork the Sia repository](#fork)
+  * [Write some code](#write)
+  * [Submit your code for review](#pull)
+  * [More git resources](#git-resources)
+* [Where to start](#where-to-start)
+* [Contact us](#contact)
+
+<a name="go"/>
+## Get started with Go
+
+<a name="install-go"/>
+### Install Go
+To install Go on your computer, follow the 
+[official installation guide][install-go].  
+
+You should install the latest [official Go binary][binary] for your system (if 
+not available, [install from source][source]).  If you plan to cross compile 
+Sia, see [Cross Compilation with Go 1.5][cross] by Dave Cheney.  
```

Comment:
`make xc` is for creating binaries for all platforms. It's more of a release tool than a cross-compilation tool. If you only want to cross-compile for a specific platform, following Dave Cheney's guide is the way to go.